ToDoList/style.css

```diff
@@ -0,0 +1,22 @@
+body{
+background-color: dimgray;
+font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+.box{
+    width:440px;
+    height:300px;
+    background: black;
+    margin-left: 100px;
+    margin-top: 100px;
+    color:aliceblue;
+    text-align-last: center;
+}
+.list{
+    color: aquamarine;
+    font-size: medium;
+    font-family: Arial, Helvetica, sans-serif;
+    font-style: italic;
+    margin-bottom: 30px;
+    line-height: 30px;
+    color: floralwhite;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/51869689/184522629-8a97957b-b250-4f64-be97-43d3da96ae17.png)
-Overflow should be hidden and a scrolling div should be implemented.
https://www.w3schools.com/css/css_overflow.asp